src/modules/add.js

```diff
@@ -14,10 +10,13 @@ const add = (e) => {
     section.insertAdjacentElement('afterend', message);
     setTimeout(() => { message.remove(); }, 2000);
   } else {
-    const score = new Scores(name, scoreV);
-
-    Interface.addScoreToList(score);
-    Store.addScore(score);
+    await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/T14uVhRrwlVGFtBvKPwB/scores/', {
+      method: 'POST',
+      body: JSON.stringify({ user: name, score: scoreV }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
```

Comment:
Good job! When the user submits the form, it's sending the data to the server, but don't you think it's a bit confusing for the user when you don't clean up the inputs and reflect the changes? It's recommended to reflect the changes when the user submits the form. So please make sure to set the value of the inputs to empty string `''` and update the table.